torch/_lobpcg.py

```diff
@@ -785,7 +785,7 @@ def update_converged_count(self):
         E, X, R = self.E, self.X, self.R
         rerr = (
             torch.norm(R, 2, (0,))
-            * (torch.norm(X, 2, (0,)) * (A_norm + E[: X.shape[-1]] * B_norm)) ** -1
+            * (torch.norm(X, 2, (0,)) * (A_norm + torch.abs(E[: X.shape[-1]]) * B_norm)) ** -1
```

Comment:
It might also be better to just divide rather than multiplying by the inverse, given that this is never going to be the bottle neck of the function.